tests/kernel/mem_protect/userspace/src/main.c

```diff
@@ -1085,6 +1085,52 @@ void test_stack_buffer(void)
 
 }
 
+void z_impl_missing_syscall(void)
+{
+	/* Shouldn't ever get here; no handler function compiled */
+	k_panic();
+}
+
+void test_unimplemented_syscall(void)
+{
+	expect_fault = true;
+	expected_reason = REASON_KERNEL_OOPS;
+
+	missing_syscall();
```

Comment:
How about when this is called from supervisor thread? We don't need a "handler" in that case, do we?

Reply:
Also. should we test this scenario?

Reply:
no, we shouldn't.
the point of this is to show that when we invoke a missing system call from user mode, the appropriate fallback function runs (handler_no_syscall handler_bad_syscall)

Reply:
thanks